backend/products/src/handlers/products.rs

```diff
@@ -22,19 +26,23 @@ pub fn configure() -> impl FnOnce(&mut ServiceConfig) {
   |config: &mut ServiceConfig| {
     let read_products_auth =
       HttpAuthentication::bearer(ValidatorBuilder::new().with_scope(PermissionName::ReadAll).build());
+
+    let create_products_auth =
+      HttpAuthentication::bearer(ValidatorBuilder::new().with_scope(PermissionName::CreateAll).build());
+
     config.service(
       web::scope(V1_PATH)
-        .wrap(read_products_auth)
         .service(get_product)
-        .service(get_products),
+        .wrap(read_products_auth.clone())
+        .service(get_products)
+        .wrap(read_products_auth.clone())
+        .service(post_products)
+        .wrap(create_products_auth),
```

Comment:
Validate this logic works, manually delete READ or CREATE access and see if endpoints work.

Reply:
resolved in 161f82e